.github/workflows/code-health.yml

```diff
@@ -28,7 +23,39 @@ jobs:
           node-version: 16.x
       - name: Install dependencies
         run: yarn install --check-files
-      - name: build
-        run: npx projen build
-
+      - name: compile
+        run: npx projen compile
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: actionlint
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+          ./actionlint -color
+        shell: bash
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: yarn install --check-files
+      - name: lint
+        run: npx projen eslint
+  dependency-review:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: Dependency Review
+        uses: actions/dependency-review-action@v3
+  shellcheck:
```

Comment:
same as above, you can split task to gain parallelism if they are not related, this is why we have files per task in the CLI https://github.com/mongodb/mongodb-atlas-cli/tree/master/.github/workflows

Reply:
they are running in parallel, all these are different jobs

Reply:
![Screenshot 2023-04-21 at 17 03 16](https://user-images.githubusercontent.com/5663078/233682785-b7ef2512-2102-433a-b036-a161d65b6c72.png)